src/Nox.Generator/Domain/Entity.template.cs

```diff
@@ -5,11 +5,33 @@
 using System;
 using System.Collections.Generic;
 
-using Nox.Types;
+using Nox.Abstractions;
 using Nox.Domain;
+using Nox.Types;
 
 namespace {{codeGeneratorState.DomainNameSpace}};
 
+/// <summary>
```

Comment:
The comments should be withing the if statement as well. Otherwise, we will have comments without domain event records where *.RaiseEvents == false.

Reply:
Desired changes were applied